components/user-menu.scss

```diff
@@ -3,6 +3,11 @@ $menu-width: 250px;
 $separator-line-size: 70%;
 
 .hdd-user-menu {
+  // When the user menu is opened the background is darker so to match this this color also need to be darker
+  @include is-dark-theme() {
+    --hdd-color-background-secondary: var(--hdd-color-grey-dark4);
```

Comment:
Denne vil overskrive --hdd-color-background-secondary for alle subelementer. Bør ha eget, unikt navn